test/integration/interactions/interaction.int.js

```diff
@@ -1,9 +1,65 @@
+/* global picassochart, triggeredEvents */
+
+const iPad = {
+  name: 'iPad landscape',
+  userAgent: 'Mozilla/5.0 (iPad; CPU OS 11_0 like Mac OS X) AppleWebKit/604.1.34 (KHTML, like Gecko) Version/11.0 Mobile/15A5341f Safari/604.1',
+  viewport: {
+    width: 1024,
+    height: 768,
+    deviceScaleFactor: 2,
+    isMobile: true,
+    hasTouch: true,
+    isLandscape: true
+  }
+};
+
 describe('picasso-interactions', () => {
-  const fixture = 'http://localhost:10001/interactions/interactions-lasso.fix.html';
+  const fixture = 'http://localhost:10001/interactions/events.fix.html';
+  const fixtureLasso = 'http://localhost:10001/interactions/interactions-lasso.fix.html';
   const fixtureRange = 'http://localhost:10001/interactions/interactions-range.fix.html';
 
+  describe('events', () => {
+    it('should be on by default', async () => {
+      await page.goto(fixture);
+      await page.waitForSelector('.container');
+      await page.click('.container');
+      const ev = await page.evaluate(() => triggeredEvents);
+      expect(ev.mousedown).to.equal(2);
```

Comment:
Slightly ode verification as you cannot be sure if an event was triggered twice or both the `on` and `native` was fired. But I do see the pain in setting up fixture to cover that case.